scripts/us_census/acs5yr/subject_tables/process.py

```diff
@@ -77,34 +91,39 @@ def main(argv):
 
     if file_extension == '.zip':
         if option == 'colmap':
-            set_column_map(input_path, spec_path, output_dir)
+            set_column_map(input_path, spec_path, output_dir, create_mcf)
         if option == 'process':
-            process_subject_tables(table_prefix=table_prefix,
-                                   input_path=input_path,
-                                   output_dir=output_dir,
-                                   column_map_path=os.path.join(
-                                       output_dir, 'column_map.json'),
-                                   spec_path=spec_path,
-                                   debug=debug,
-                                   delimiter='!!',
-                                   has_percent=has_percent)
+            if not table_prefix:
+                print('table_prefix required to run data loader')
```

Comment:
s/data loader/next steps of data processing.

Reply:
didn't get this, can you explain in detail?

Reply:
replace `data loader` with the phrase `next steps of data processing.`